database/encdb/db_test.go

```diff
@@ -22,3 +22,18 @@ func TestInterface(t *testing.T) {
 		test(t, db)
 	}
 }
+
+func BenchmarkInterface(b *testing.B) {
```

Comment:
In all of the `BenchmarkInterface` functions, are we intending to use the same database for each size?

Reply:
Yes, the idea was for each size to represent the byte size of values read or written for each database implementation.

Reply:
Sorry I might not have been clear. What I meant was that the same _instance_ of the database is used for each key size. So when we benchmark the `32` bytes size, the database is empty. Then when we benchmark the `64` bytes size the database already has the `32` bytes size data inside of it, ect.

Reply:
Fixed such that every db is renewed. I'm not sure the `rpcdb` `db_test.go` renews the database in the right way however.